aws_cloudtrail_rules/aws_security_configuration_change.py

```diff
@@ -1,26 +1,29 @@
+from panther_base_helpers import deep_get
+
 SECURITY_CONFIG_ACTIONS = {
-    'DeleteAccountPublicAccessBlock',
-    'DeleteDeliveryChannel',
-    'DeleteDetector',
-    'DeleteFlowLogs',
-    'DeleteRule',
-    'DeleteTrail',
-    'DisableEbsEncryptionByDefault',
-    'DisableRule',
-    'StopConfigurationRecorder',
-    'StopLogging',
+    "DeleteAccountPublicAccessBlock",
+    "DeleteDeliveryChannel",
+    "DeleteDetector",
+    "DeleteFlowLogs",
+    "DeleteRule",
+    "DeleteTrail",
+    "DisableEbsEncryptionByDefault",
+    "DisableRule",
+    "StopConfigurationRecorder",
+    "StopLogging",
```

Comment:
Curious why this was changed to `"` while a previous one changed to `'` ? something the linter did?

Reply:
Mainly for consistency

Reply:
So now that we're using Black as our formatter, double quotes are religiously enforced (exceptions for f-strings)